src/native/eventpipe/ep.h

```diff
@@ -320,6 +320,28 @@ EP_WRITE_BUFFER_INT (64, u)
 
 #undef EP_WRITE_BUFFER_INT
 
+#if SIZEOF_VOID_P == 4
```

Comment:
Couldn't this be a ep_write_buffer_uintptr_t setup similar as the macros, we already have a ep_rt_val_uintptr_t where we take care of of size, so better to just have this in one place than spread out, something like (or fix macro to be a little more flexible):

```
static
inline
void
ep_write_buffer_uintptr_t (uint8_t **buffer, uintptr_t value)
{
	value = ep_rt_val_uinptr_t (value);
	memcpy (*buffer, &value, sizeof (value));
	*buffer += sizeof (value);
}
```